### PR TITLE
Made smartDashboard changes

### LINKED ISSUE #66: Add Smart Dashboard Stuff

### PR DESCRIPTION
**Pull request by Jacob4649**
Fix #66

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Revolver.java [hunk 1]
```diff
@@ -20,6 +21,8 @@
   private final CANSparkMax m_revolver;
   private int m_drumReps = 0;
 
+  private UnjamRevolver m_unjam = new UnjamRevolver(this);
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Revolver.java]: `final`, and move init to constructor for consistency

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 2]
```diff
@@ -105,4 +106,11 @@ public void useOutput(double output, double setpoint) {
   public double getMeasurement() {
     return getAverageSpeed();
   }
+
+  @Override
+  public void periodic() {
+    SmartDashboard.putNumber("shooter/speed", getAverageSpeed());
+    SmartDashboard.putNumber("shooter/setpoint", getSetpoint());
+    SmartDashboard.putNumber("shooter/percentSpeed", getAverageSpeed()/getSetpoint());
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Shooter.java]: percentSetpoint? Not sure. Not a big idea

File: src/main/java/frc/robot/subsystems/Turret.java [hunk 3]
```diff
@@ -103,4 +104,9 @@ private double sensorUnitsToDegrees(int raw) {
   private int degreesToSensorUnits(double degrees) {
     return (int) Math.round(degrees / (Constants.Turret.POT_SCALE * Constants.Turret.RATIO) * 1023d);
   }
+
+  @Override
+  public void periodic() {
+    SmartDashboard.putNumber("turret/angle", getAngle());
```
Comment (matthew-reynolds) [inline src/main/java/frc/robot/subsystems/Turret.java:110]: Add setpoint

**Review by matthew-reynolds — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Turret.java (hunk 3 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Turret.java:110]: resolved in f2e6a54

File: src/main/java/frc/robot/subsystems/Shooter.java (hunk 2 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Shooter.java]: resolved in f2e6a54

File: src/main/java/frc/robot/subsystems/Revolver.java (hunk 1 above)
Comment (Jacob4649) [reply src/main/java/frc/robot/subsystems/Revolver.java]: resolved in f2e6a54

**Review by matthew-reynolds — APPROVED** [review]

**Pull request merged**